### PR TITLE
style(): format codebase with dotnet-format

### PR DESCRIPTION
**Pull request by smolck**
Formats the codebase with [dotnet-format](https://github.com/dotnet/format).

Note that this *is* the official formatter for .NET 6, as it's builtin to the SDK (see the pinned issue in the linked repo above), but until that's released it has to be installed via [the Nuget package](https://www.nuget.org/packages/dotnet-format/).

### CODE REVIEW

File: src/NvimClient.APIGenerator/NvimAPIGenerator.cs [hunk 1]
```diff
@@ -140,7 +140,8 @@ private object GetExtensionType(MessagePackExtendedTypeObject msgPackExtObj)
   public class {eventName}EventArgs : EventArgs
   {{
 {
-      string.Join("", uiEvent.Parameters.Select(param => {
+      string.Join("", uiEvent.Parameters.Select(param =>
+      {
```
Comment (justinmk) [inline src/NvimClient.APIGenerator/NvimAPIGenerator.cs:144]: sigh, are they really forcing this? would be nice to have opening braces at EOL.

**Review by justinmk — APPROVED** [review]

File: src/NvimClient.APIGenerator/NvimAPIGenerator.cs (hunk 1 above)
Comment (smolck) [reply src/NvimClient.APIGenerator/NvimAPIGenerator.cs:144]: I can check if there's an option for it, but yes, afaik that is the idiomatic formatting used in C#.

Comment (smolck) [reply src/NvimClient.APIGenerator/NvimAPIGenerator.cs:144]: Okay so there *is* a config option for this (`csharp_new_line_before_open_brace = none` in `.editorconfig`). Want me to set that?

**Comment by smolck** [comment]
I’ve gone ahead and merged this; if we want to use `csharp_new_line_before_open_brace = none` let me know and I can make another PR. I think I’d tend to lean toward keeping the new lines though, as that’s the most often way of doing it in C#, afaik. At least that’s the way [.NET/Visual Studio does their formatting (see point 1)](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md) fwiw.

**Comment by justinmk** [comment]
> At least that’s the way [.NET/Visual Studio does their formatting (see point 1)](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md) fwiw.

Ok. When I read the .net source years ago I thought they did opening braces at EOL. But I see it's not the case anymore. https://github.com/dotnet/runtime/blob/1115c43390fdaa232d082114be1bb4eb79387a71/src/libraries/Common/src/System/HexConverter.cs

**Pull request merged**